scripts/prepare-env.js

```
@@ -0,0 +1,53 @@
+function run() {
+	let envConfig = {};
+
+	switch (DEPLOYMENT_ENV) {
+		case 'production':
+			envConfig = loadEnv(path.join(envDir, '.env.production'));
+			break;
+		case 'development':
+			envConfig = loadEnv(path.join(envDir, '.env.development'));
+			break;
+		case 'local':
+		default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)